src/fsharp/FSharp.Core/list.fs

```diff
@@ -549,27 +562,27 @@ namespace Microsoft.FSharp.Collections
         let tryFindIndexBack predicate list = list |> toArray |> Microsoft.FSharp.Primitives.Basics.Array.tryFindIndexBack predicate
 
         [<CompiledName("Sum")>]
-        let inline sum          (list:list<'T>) =
+        let inline sum (list:list<'T>) =
             match list with
-            | [] ->  LanguagePrimitives.GenericZero< 'T >
+            | [] -> LanguagePrimitives.GenericZero< 'T >
```

Comment:
I think <'T> is preferable to < 'T >

Reply:
I believe @dsyme recently changed it to the one with spaces. So I didn't want to touch it. But not sure

Reply:
Elsewhere in the same file and throughout the rest of the codebase it's <'T'> so we should stick with that.

Reply:
ok done.